### ISSUE TITLE
could retrain from model.acc.best or model.loss.best

### ISSUE THREAD

**Issue by qmpzzpmq** [espnet/espnet#1591]
Is there any method resume from model.acc.best or model.loss.best and using a opt from the setting conf/train.yaml
if there is no this method and your guys want it, I can add this function.

**Comment by b-flo** [comment]
Hi,

> Is there any method resume from model.acc.best or model.loss.best and using a opt from the setting conf/train.yaml

You mean loading a pre-trained model and changing the optimizer? If so, you could use these options: https://github.com/espnet/espnet/blob/master/espnet/bin/asr_train.py#L179-L189

**Comment by windtoker** [comment]
Hi. Just wondering if its also available in tts module as well. found out what you mentioned is defined in asr_train.py but not in tts_train.py and also as a background defined in asr.py but not in tts.py.  Im asking to confirm if there would be any problem applying that fine tuning part to tts module as well.

**Comment by b-flo** [comment]
Yes, these methods are defined for ASR/MT/ST. There is no equivalent for TTS that I'm aware of.
Maybe you have some comments @kan-bayashi ?

**Comment by kan-bayashi** [comment]
OK. I will implement it!

**Comment by kan-bayashi** [comment]
I implemented the function to load pretrained model for TTS in #1536 
I will close this issue.

**Comment by windtoker** [comment]
Thank you for the implementation.

**Comment by marcosnetopires** [comment]
Sorry, but how exactly do I resume from model.acc.best? what should I add to the ./run.sh command to make this work?
